modules/ROOT/pages/api-extensions.adoc

```diff
@@ -0,0 +1,462 @@
+= Anypoint DataGraph API Extension Reference
+
+The following guide defines API extensions DataGraph. 
+
+There are two groups of extensions: Entity extensions and DataGraph extensions. You must represent each group as different extension instances and store them in Exchange as different assets.
+
+== Entity Extensions
```

Comment:
Right now we decided to use a single group, datagraph extensions, instead of two.